docs/cli/Guides/collaboration.md

```
@@ -57,23 +56,23 @@ sequenceDiagram
+Alice builds a <a id="solution"><span className="dashed-underline">solution</span></a>—a Docker image containing her script ([1](/cli/guides/collaboration#alice-1-build-a-solution)). She uploads the solution using SPCTL ([2](/cli/guides/collaboration#alice-2-upload-the-solution)) and grants Bob access for verification ([3](/cli/guides/collaboration#alice-3-send-the-solution-to-bob)).
 
-Bob (or an independent auditor) downloads the solution (4) and verifies that it is safe to process his data (5).
+Bob (or an independent auditor) downloads the solution ([4](/cli/guides/collaboration#bob-4-download-the-solution)) and verifies that it is safe to process his data ([5](/cli/guides/collaboration#bob-5-verify-the-solution)).
 
-Bob uploads his dataset to remote storage using SPCTL (6). The dataset is automatically encrypted during upload, and only Bob holds the key.
+Bob uploads his dataset to remote storage using SPCTL ([6](/cli/guides/collaboration#bob-6-upload-the-dataset)). The dataset is automatically encrypted during upload, and only Bob holds the key.
 
-Bob creates an <a id="offer"><span className="dashed-underline">offer</span></a> on the Marketplace (7). The offer requires Bob's manual approval for use. He shares the offer's IDs with Alice.
+Bob creates an <a id="offer"><span className="dashed-underline">offer</span></a> on the Marketplace ([7](/cli/guides/collaboration#bob-7-create-an-offer)). The offer requires Bob's manual approval for use. He shares the offer's IDs with Alice.
 
 **Execution**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)